markdown/3.x/en/faq.md

```diff
@@ -3,7 +3,21 @@
 #### Question:
 I want to use FACT-Finder Web Components with Angular, but data binding used in the components conflicts with Angular's `{{ }}` binding. Can you fix it?
 #### Answer:
-Default binding delimiters can be changed via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine).
+You can add Angular's `ngNonBindable` attribute to the parent element to prevent Angular from parsing double curly braces. If you have to keep both Angular and FACT-Finder WebComponents bindings within the same element, you can use: 
+```html
+<ng-container ngNonBindable>
+    {{FF Binding}}
+</ng-container>
+```
+The `<ng-container>` tag will not be rendered in the DOM.
+
+Alternatively, you can change the default FACT-Finder Web Components binding delimiters via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine). Please note that HTML binding will still require a single curly bracket, what will cause an Angular error. You can utilize `ngNonBindable` to handle this issue as well:
```

Comment:
I would add information about FF Web Components version that supports custom Mustache delimiters. The documentation is (or at least should be) for the latest one, we cannot assume that the users update their Web Components with every release.

Reply:
We don't do it for other features we implement. I don't think we need it here.

Reply:
This FAQ answer is valid only for version 3.6.0 or above. I don't expect users with 3.5.0 or below who try this will check CHANGELOG.md to verify if their version supports this feature. They will rather complain and report "not working". If we don't do it for other features it doesn't mean it is fine not to do it.

Reply:
I won't, we should be consistent. If we start putting a note with a version for every single feature we implement the documentation will look terrible.

Reply:
We could add another FAQ entry:
Q: Which version is the documentation referring to?
A: Latest

Reply:
@omikron-fp That's actually a great idea.